### PR TITLE
vim-patch:8.2.{70,74,75,76}

### PR DESCRIPTION
**Pull request by janlazo**

### CODE REVIEW

File: src/nvim/options.lua [hunk 1]
```diff
@@ -2488,12 +2488,6 @@ return {
       varname='p_tbidi',
       defaults={if_true={vi=false}}
     },
-    {
-      full_name='termencoding', abbreviation='tenc',
-      type='string', scope={'global'},
-      vi_def=true,
-      defaults={if_true={vi=""}}
-    },
```
Comment (justinmk) [inline src/nvim/options.lua]: we had tentative plans to restore this, or at least detect it internally for converting  input/output to/from the terminal. https://github.com/neovim/neovim/issues/7445

Comment (blueyed) [reply src/nvim/options.lua]: Removing it causes problems with plugins, e.g. vim-gitgutter: https://github.com/airblade/vim-gitgutter/pull/671

Comment (blueyed) [reply src/nvim/options.lua]: This should likely get handled like with gvim, where it is forced to be "utf-8" always.

Comment (justinmk) [reply src/nvim/options.lua]: Yes, that's why we left it in, but I'm not really sure that's a good reason. If we don't support options we should just remove them, generally...

Comment (justinmk) [reply src/nvim/options.lua]: > This should likely get handled like with gvim, where it is forced to be "utf-8" always.

I just noticed that as well. Hmm.

Comment (janlazo) [reply src/nvim/options.lua]: Restored it in https://github.com/neovim/neovim/pull/11662

**Pull request merged**